examples/mongodb-atlas/mongodb-atlas-blueprint.yaml

```
@@ -0,0 +1,205 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: mongodb-atlas-blueprint
+actions:
+  backup:
+    outputArtifacts:
+      backupOutput:
+        keyValue:
+          snapshotId: "{{ .Phases.mongoBackup.Output.snapshotId }}"
+    phases:
+    - func: KubeTask
+      name: mongoBackup
+      args:
+        namespace: "{{ .Object.metadata.namespace }}"
+        image: docker.io/pratikrshah/mongo-atlas:0.90.0-test1
+        command:
+          - bash
+          - -o
+          - errexit
+          - -o
+          - pipefail
+          - -c
+          - |
+            org_id=$(echo '{{ .Object.data.orgid | toString }}' | base64 --decode)
```

I was talking about `b64dec` [sprig](http://masterminds.github.io/sprig/encoding.html) function last night. If you use that, this template will look like this:
```suggestion
            org_id="{{ .Object.data.orgid | toString | b64dec }}"
```
Did you try it out?
We have used this before in `etcd-blueprint`

---

I was looking into different function. this worked.